crates/bevy_ecs/src/world/error.rs

```diff
@@ -1,12 +1,62 @@
-//! Contains error types returned by bevy's schedule.
+//! Contains error types returned by methods on [`World`].
+//!
+//! [`World`]: crate::world::World
+
+use std::any::TypeId;
 
 use thiserror::Error;
 
-use crate::schedule::InternedScheduleLabel;
+use crate::{component::ComponentId, prelude::*, schedule::InternedScheduleLabel};
```

Comment:
Just noticed that these imports should probably be scoped to `#[cfg(feature = "bevy_reflect")]` as well, because they will throw "unused import" warnings otherwise, I'd think.
Should I do that or how is it usually handled?

Reply:
It's been moved to the `world::reflect` module as well, which is `cfg(feature = "bevy_reflect")`, so this should also be fixed now!